data/posts/2022-11-04-considerations-before-first-nomad-trip.md

```diff
@@ -0,0 +1,77 @@
+
+### Time Zones
+
+The time zone of the location you are working in is a huge consideration. Some nomads prefer working in the same or very similar time zones to where their other colleagues are based. Others don't mind working further and making moderate to relatively large adjustments in their sleep cycle and structure of their day. It’s often a good idea to consider working in areas that will allow you to overlap for at least a few hours per day so that you will have some time to collaborate with your colleagues in real-time when needed. 
+
+## Lifestyle
+I would recommend giving some thought to what kind of lifestyle you'll want to live while abroad. In order to do so, at a minimum, you'll need to consider the cost of living of places you are interested in, preferences for your accomodation and working, ease of access to service and goods, and logistics for traveling domestically or otherwise in destinations of interest to you. 
+
+### Cost of Living
+
+ A major factor in how you'll live while abroad is the cost of living. When I plan a trip, a lower cost of living is often a draw for me. Although I don't entirely avoid higher cost of living locations, I definitely stay there for less time than I do less expensive locations. When planning a trip you should take a look at the cost of living for the specific cities or towns you're curious about and pay attention to average prices for food and lodging in particular. 
+
+### Your Accomodation and Workplace
+
+I would definitely recommend finding out where the more central neighborhoods are where you're staying and trying to stay in one of those if they are considered to be safe. This generally costs a bit more, but allows you to make more of the time you have there. Staying somewhere that is a bit further out, but well connected by public transit is also something to consider, especially if you're going to be staying awhile. I really prefer to rent an apartment and often work from there, but I don't think everyone should necessarily do that -- many nomads really enjoy both living in and working in shared accomodations.
+
+Some hostels or more communal accomodations do have quieter common areas with desk space where you could work if that's an option that is interesting to you. As an example, I once stayed at Cityhub in Copenhagen and was able to get work done in the shared area near the kitchen with no problem. Additionally, if you have a different working preference than me and really prefer to get out of the house, you can try to look into local options such as cafes. If you think you may want to often do work at cafes, try to find out ahead of time if outlets are available at the cafes you are interested in. Another option, especially common in digital nomad hubs, is to work at local co-working places. This will allow you to have space to work and also meet other people who are working while traveling as well
```

Comment:
```suggestion
Some hostels or more communal accommodations do have quieter common areas with desk space where you could work if that's an option that is interesting to you. As an example, I once stayed at Cityhub in Copenhagen and was able to get work done in the shared area near the kitchen with no problem. Additionally, if you have a different working preference than me and really prefer to get out of the house, you can try to look into local options such as cafes. If you think you may want to often do work at cafes, try to find out ahead of time if outlets are available at the cafes you are interested in. Another option, especially common in digital nomad hubs, is to work at local co-working places. This will allow you to have space to work and also meet other people who are working while traveling as well
```